pkg/subctl/cmd/gather/resource.go

```diff
@@ -26,27 +27,32 @@ import (
 )
 
 func ResourcesToYAMLFile(info *Info, ofType schema.GroupVersionResource, namespace string) error {
-	list, err := info.DynClient.Resource(ofType).Namespace(namespace).List(metav1.ListOptions{})
+	lists, err := info.DynClient.Resource(ofType).Namespace(namespace).List(metav1.ListOptions{})
```

Comment:
```suggestion
	list, err := info.DynClient.Resource(ofType).Namespace(namespace).List(metav1.ListOptions{})
```
It is a single list of multiple items. `lists` makes it confusing.